src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPay = 40;
+  let discount = 0;
+
+  if ((days >= 3) && (days <= 6)) {
+    discount = 20;
+  } else if (days >= 7) {
+    discount = 50;
+  }
```

Comment:
you may use 2 conditions (and 2 if's respectively):
* `>= 3` and
* `>= 7`

Reply:
Bohdan, thank you for your recommendations! 👍
Well noted! Will act accordingly!